### PR TITLE
start of like feature

### PR DESCRIPTION
**Pull request by ashleygranitto**
After I finished the six data steps last week, I began working on a like feature such that every time a comment's heart icon was clicked, its like value was updated by one. So far, I have the styling complete. I also have figured out how to update an entity (you pretty much have to recreate it reusing the same ID). This commit includes my code so far--right now, a user can click the icon as many times as they want and it will update. At the end of this week after implementing authentication and all other steps required, I want to revisit this feature and ensure an authenticated user cannot like a comment more than once (and also allow them to unlike a comment).

### CODE REVIEW

File: portfolio/src/main/webapp/comment-form.js [hunk 1]
```diff
@@ -22,19 +22,62 @@ async function postFeed(quantity) {
...
+    secElement.appendChild(addComment(comment));
+    secElement.appendChild(addLoveFeature(comment));  
+    container.appendChild(secElement);
   });
 }
 
+/** Return a paragraph element that contains a user comment */
+function addComment(comment) {
+  const pElement = document.createElement('p');
+  console.log(comment);
```
Comment (win-chen) [inline portfolio/src/main/webapp/comment-form.js]: remove logs

File: portfolio/src/main/webapp/comment-form.js [hunk 2]
```diff
@@ -22,19 +22,62 @@ async function postFeed(quantity) {
...
+  // TO DO--determine using Datastore how many likes to diplay--0 is placeholder for now
+  const counter = document.createElement('p');
+  counter.innerText = comment.count;
+  return counter; 
+}
+
 /** Delete every comment from the feed */
 async function deleteComments() {
-  // The fetch will delete every comment and return an empty string
+  // response is not meaningful and is just used to catch the fetch return
```
Comment (win-chen) [inline portfolio/src/main/webapp/comment-form.js]: You don't need to set the await result to anything. You can do:
await fetch();
...rest of function
And the rest of the function won't run until the await has completed.

File: portfolio/src/main/webapp/comment-form.js [hunk 3]
```diff
@@ -22,19 +22,62 @@ async function postFeed(quantity) {
...
   const response = await fetch("/delete-data", { method: 'POST' });
-  feed.innerHTML = response;
+  feed.innerHTML = '';
+}
+
+/** Increment a comment's like value by one */
+async function updateLike(comment) {
+  // response is not meaningful and is just used to catch the fetch return
+  const response = await fetch('/update-count?id=' + comment.id + '&count=' + comment.count
```
Comment (win-chen) [inline portfolio/src/main/webapp/comment-form.js]: For a complex string like this you can use template strings
https://google.github.io/styleguide/jsguide.html#features-strings-template-strings

**Review by win-chen — CHANGES_REQUESTED** [review]

**Review by mmallett — APPROVED** [review]
LGTM, make sure you give yourself plenty of time to work on this week's walkthroughs

File: portfolio/src/main/webapp/comment-form.js (hunk 1 above)
Comment (ashleygranitto) [reply portfolio/src/main/webapp/comment-form.js]: removed

File: portfolio/src/main/webapp/comment-form.js (hunk 2 above)
Comment (ashleygranitto) [reply portfolio/src/main/webapp/comment-form.js]: oh that's much better--adjusted

File: portfolio/src/main/webapp/comment-form.js (hunk 3 above)
Comment (ashleygranitto) [reply portfolio/src/main/webapp/comment-form.js]: I ended up splitting it into two different template strings--I'm not sure if I used them correctly but I'll include on my next commit!

**Review by win-chen — APPROVED** [review]

**Pull request merged**